src/net/java/sip/communicator/impl/protocol/irc/properties/IrcProperties.java

```diff
@@ -0,0 +1,15 @@
+package net.java.sip.communicator.impl.protocol.irc.properties;
+
+/**
+ * Properties for the IRC plugin
+ *
+ * @author Duncan Robertson
+ */
+public class IrcProperties
+{
+    public static final String PROP_IRC_IGNORE = "net.java.sip.communicator.impl.protocol.irc.properties.IGNORE_STRING";
+
+    public IrcProperties()
```

Comment:
This constructor is redundant. If at all, make it private to mark IrcProperties as a utility class.